Trakt/Trakt.csproj

```diff
@@ -1,9 +1,11 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0;</TargetFrameworks>
     <AssemblyVersion>3.3.0.0</AssemblyVersion>
     <FileVersion>3.3.0.0</FileVersion>
+    <AssemblyVersion>3.2.2.0</AssemblyVersion>
```

Comment:
wtf, diff shows two `AssemblyVersion` and `FileVersion` fields?

Reply:
I don't know. Rider only shows the 3.2.2.0 one :/